### PR TITLE
Consider manifest to be empty if the corresponding manifest file does not exist

### LINKED ISSUE #29: Errno::ENOENT No such file or directory manifest.json
For some applications where I use WDS, WDS is capable of serving all assets (its manifest contains all entries I need to use), so it's not necessary to generate the disk-based manifest file. On a fresh clone of my project, the on-disk manifest file doesn't exist.

This leads to the following unexpected scenario:

1. Start WDS
1. Try to include (via the asset helpers) an asset that does not exist

Expected:

`ExternalAssetPipeline::AssetNotFound` is raised telling me that the asset doesn't exist

Observed: 

`Errno::ENOENT No such file or directory @ rb_sysopen - /Users/chris.larose/dev/mycase/login/public/assets/manifest.json`

### PR DESCRIPTION
**Pull request by cjlarose**
Fixes #29

### CODE REVIEW

File: lib/external_asset_pipeline/manifest.rb [hunk 1]
```diff
@@ -29,6 +29,8 @@ def data
 
     def load
       JSON.parse(@config.manifest_path.read)
+    rescue Errno::ENOENT
+      {}
```
Comment (rmacklin) [inline lib/external_asset_pipeline/manifest.rb]: We have this warning when a `ServerManifest` is used but the corresponding server isn't `running?`:
https://github.com/rails-front-end/external_asset_pipeline/blob/2bbf45e615e48dc8afd5fbcb7b0f0bce0c761633/lib/external_asset_pipeline/server_manifest.rb#L27-L29

Do you think it's worth logging a similar warning in this scenario?

Comment (rmacklin) [reply lib/external_asset_pipeline/manifest.rb]: Alternatively, we could add a `Configuration` attribute that defaults to `true` but can be set to `false` in the development environment, and this code could re-`raise` the exception if that attribute is `true`

Comment (cjlarose) [reply lib/external_asset_pipeline/manifest.rb]: Done!

**Comment by rmacklin** [comment]
This has been released in [`v0.7.0`](https://rubygems.org/gems/external_asset_pipeline/versions/0.7.0)!

**Pull request merged**